xml/System/RuntimeMethodHandle.xml

```diff
@@ -251,7 +251,7 @@
         <remarks>
           <format type="text/markdown"><![CDATA[  
 ## Remarks  
-   The returned <xref:System.IntPtr> is meant for internal runtime usage and should not be used to call the function from native code. Use <xref:System.Runtime.InteropServices.Marshal.GetFunctionPointerForDelegate%2A?displayProperty=nameWithType> in order to get a function pointer that can be passed to native code.  
+   If the method this handle represents is static, then the returned value can be cast to a C# managed function pointer type with the same signature. If the method has the <xref:System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute%2A?displayProperty=nameWithType> attribute, then the returned value is an unmanaged function pointer with the same calling convention as specified in the attribute. For all other cases, the value is not easily usable from user code and is meant exclusively for usage within the runtime. For methods that do not have the <xref:System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute%2A?displayProperty=nameWithType>, use <xref:System.Runtime.InteropServices.Marshal.GetFunctionPointerForDelegate%2A?displayProperty=nameWithType> in order to get a function pointer that can be passed to native code.  
```

Comment:
Is "for all other cases" different to "for methods that don't have the UnmanagedCallersOnlyAttribute"? If so, then I think the wording needs to be clarified for what "all other cases" refers to.